src/gui/mzroll/peakdetectiondialog.cpp

```diff
@@ -157,6 +158,11 @@ PeakDetectionDialog::PeakDetectionDialog(QWidget *parent) :
 }
 
 
+void PeakDetectionDialog::onReset()
```

Comment:
@rish9511 Why an extra layer of signal and slots?

Reply:
@sahil21 how is this an extra layer ?

Reply:
@rish9511 Sorry my bad. Ignore.